vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -730,9 +744,9 @@ - (void)reportPostData:(NSData *)postData url:(NSURL *)url tabId:(UInt32)tabId f
                           std::string(url.absoluteString.UTF8String),
                           tabId,
                           "", "", "", "");
-
+  
```

Comment:
looks like you might have something weird going on in your editor that is adding whitespace?

Reply:
Yeah Xcode doesn't trim whitespace-only lines